toolkit/templates/character_generator.html

```diff
@@ -3,4 +3,651 @@
 {% load static %}
 
 {% block application_pane %}
+  <div class="character-header">
+    <img src
+         {% static "Character_Header.png" %}
```

Comment:
Ohhh, we got _pictures_ in our generator now?

Reply:
👀

Reply:
![image](https://user-images.githubusercontent.com/45466247/193432250-27fc2c62-e574-49ce-840d-715ef6d464af.png)

Pictures are not loading for some reason